### PR TITLE
Fix "invalid syntax" issue with discordgo

### PR DESCRIPTION
**Pull request by Augiro**
I encountered the #154 issue, and tried upgrading the discordgo library to the latest version, as the issue is likely caused by Discord introducing some API breaking change (which somehow doesn't always break discordgo?).

I've never seen the issue happen since upgrading discordgo, and the only issue I encountered by upgrading the discordgo version was a type that needed to be changed for permissions (int -> int64).

My IDE also complained about there being known vulnerabilities in the version used of the crypto library, so I upgraded that as well.

TLDR;
Upgrade versions of discordgo and crypto dependencies.

### CODE REVIEW

**Comment by kidd0x** [comment]
> I encountered the #154 issue, and tried upgrading the discordgo library to the latest version, as the issue is likely caused by Discord introducing some API breaking change (which somehow doesn't always break discordgo?).
> 
> I've never seen the issue happen since upgrading discordgo, and the only issue I encountered by upgrading the discordgo version was a type that needed to be changed for permissions (int -> int64).
> 
> My IDE also complained about there being known vulnerabilities in the version used of the crypto library, so I upgraded that as well.
> 
> TLDR; Upgrade versions of discordgo and crypto dependencies.

sup, can you give the fixed verion to just download it as are the releases? i can't figure it out how to compile your fixed version

**Comment by rumblefrog** [comment]
Thanks!

**Comment by kidd0x** [comment]
@Augiro 
Hey man, so i encounter another problem when im trying to use r/sendchannel or r/receivechannel
image
